packages/vite-plugin-tempest/src/config.ts

```diff
@@ -0,0 +1,22 @@
+import type { TempestViteConfiguration } from './types'
+import { exec, php } from './utils'
+
+const TEMPEST_BIN = 'tempest'
+const VITE_CONFIG_COMMAND = 'vite:config'
+
+export async function loadTempestConfiguration(): Promise<TempestViteConfiguration> {
+	try {
+		const { stdout } = await exec(`${php.value} ${TEMPEST_BIN} ${VITE_CONFIG_COMMAND}`)
```

Comment:
What's the advantage to getting this information with an `exec` call over having Tempest write a config `json` file on boot?

Reply:
This puts the burden on fetching the information on the plugin, instead of it being on Tempest.

If someone's not using Vite, they would not need this file, too. But more importantly, Vite will ask for it when it needs it (once at build time, and every time the dev server starts). Otherwise, Tempest would write the file for each request _and_ we would have to find a way to update that information on the Vite side, since the server would have already been started.